sdk/include/aditof/camera_definitions.h

```diff
@@ -56,9 +56,16 @@ struct CameraDetails {
     ConnectionType connection;
 
     /**
-     * @brief The range of the camera in mm for the operating mode
+     * @brief The maximum extremity of the range of the camera in mm for the
+     * operating mode
      */
-    int range;
+    int rangeMax;
```

Comment:
how about maxDepth and minDepth? It's more descriptive, makes it clear that this is for depth and not IR.

Reply:
done, thanks